kuksa_databroker/databroker-cli/src/main.rs

```diff
@@ -116,8 +116,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         let ca_cert = tonic::transport::Certificate::from_pem(pem);
 
         let tls_config = tonic::transport::ClientTlsConfig::new()
-            .ca_certificate(ca_cert)
-            .domain_name("Server");
+            .ca_certificate(ca_cert);
+            // With new tonic we do not need the hack of setting Server as domain name,
+            // it can resolve both localhost and 127.0.0.1 as subjectAltName
+            //.domain_name("Server");
```

Comment:
While informative for this PR, the comment should probably not be kept in the code going forward (i.e. should not be here).

Reply:
I can just close that PR as it would be just as good to include that it in this one (as it's TLS related anyway).

Reply:
Closed the PR. Also, when you remove this comment, run `cargo fmt` as the build is failing because of faulty formatting (at these lines).